src/benchmarks/beluga_vs_nav2/Earthfile

```
@@ -34,6 +34,7 @@ devel:
         apt clean && rm -rf /var/lib/apt/lists/*
     RUN pip install linuxdoc sphinxcontrib.datatemplates sphinxcontrib-repl
     DO os+ADDUSER --user=${user} --uid=${uid} --gid=${gid} --workdir=/workspace
+    DO os+FIXUSER
```

@marcoshuck +1 to moving it to a `rosdep` folder, but doesn't it mean now we have to import it? Also, shouldn't this be:

```suggestion
    DO rosdep+FIXUSER
```
?

---

Good call, I'll rebuild it once more :)

---

It's building as intended